faster_whisper/transcribe.py

```diff
@@ -642,6 +647,10 @@ def generate_with_fallback(
         max_initial_timestamp_index = int(
             round(options.max_initial_timestamp / self.time_precision)
         )
+        if options.max_length is not None:
+            max_length = min(self.max_length, len(prompt) + options.max_length)
```

Comment:
@metame-none, hello. Why do you need to get min in this logic ? If we use max_length > default (448 for faster whisper), will it cause too negative effects ? If not, I think can make this logic simpler by removing the min function.